on_board_main.ipynb

```diff
@@ -0,0 +1,48 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Code to run pynq board\n",
+    "=======\n",
+    "\n",
+    "This code act as the main function file that connect our HDC code with the pynq board so that we can run the trainings"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pynq\n",
+    "from pynq import Overlay\n",
+    "\n",
+    "overlay = Overlay(\"../\")\n",
```

Comment:
Based off my very basic research on the overlay class, this should be getting a bit file. Are we leaving this blank and updating once we have that bit file?

Reply:
yeah